src/dymaptic.GeoBlazor.Core/Scripts/arcGisJsInterop.ts

```diff
@@ -1075,4 +1110,32 @@ function waitForRender(viewId: string, dotNetRef: any): void {
             }
         }, 100);
     })
+}
+
+
+
+function buildDotNetListItem(item: ListItem): DotNetListItem | null {
+    if (item === undefined || item === null) return null;
+    let children: Array<DotNetListItem> = [];
+    item.children.forEach(c => {
+        let child = buildDotNetListItem(c);
+        if (child !== null) {
+            children.push(child);
+        }
+    });
+
+    /// <summary>
+    /// The Action Sections property and corresponding functionality will be fully implemented
+    /// in a future iteration.  Currently, a user can view available layers in the layer list widget
+    /// and toggle the selected layer's visiblity. More capabilities will be available after full
+    /// implementation of ActionSection.
+    /// </summary>
+
+    return {
+        title: 'title',
+        layer: item.layer,
+        visible: item.visible,
+        children: children,
+        actionSections: item.actionsSections
```

Comment:
Likewise, I still believe that `item.actionsSections` here will always be null or empty, bc there's been no chance to populate it. It might be worth putting a breakpoint there to see if I'm wrong.